R/eval-walk.R

```diff
@@ -101,6 +107,8 @@ ensure_named <- function(pos,
     return(set_names(pos, NULL))
   }
 
+  check_empty(pos, allow_empty, call = call)
+
```

Comment:
Does the check need to be before this early return?

Reply:
Good catch, thanks.